apps/financial-aid/web-veita/src/routes/Supervisors/supervisors.tsx

```diff
@@ -0,0 +1,189 @@
+  formatNationalId,
+  Staff,
+  StaffRole,
+  staffRoleDescription,
+} from '@island.is/financial-aid/shared/lib'
+import {
+  SupervisorsQuery,
+  UpdateStaffMutation,
+} from '@island.is/financial-aid-web/veita/graphql'
+import { AdminContext } from '@island.is/financial-aid-web/veita/src/components/AdminProvider/AdminProvider'
+
+export const Supervisors = () => {
+  const [isModalVisible, setIsModalVisible] = useState(false)
+  const [supervisors, setSupervisors] = useState<Staff[]>()
+  const { admin } = useContext(AdminContext)
+
+  const [getSupervisors, { data, error, loading }] = useLazyQuery<{
+    supervisors: Staff[]
+  }>(SupervisorsQuery, {
+    fetchPolicy: 'no-cache',
+    errorPolicy: 'all',
+  })
+  const [updateStaff, { loading: staffLoading }] = useMutation(
+    UpdateStaffMutation,
+  )
+
+  useEffect(() => {
+    getSupervisors()
+  }, [])
+
+  useEffect(() => {
+    if (data?.supervisors) {
+      setSupervisors(data.supervisors)
+    }
+  }, [data])
+
+  const isLoggedInUser = (staff: Staff) =>
+    admin?.nationalId === staff.nationalId
+
+  const changeUserActivity = async (staff: Staff) => {
+    return await updateStaff({
+      variables: {
+        input: {
+          id: staff.id,
+          active: !staff.active,
+        },
+      },
+    })
+      .then(() => {
+        getSupervisors()
+      })
+      .catch(() => {
+        toast.error(
+          'Það mistókst að breyta hlutverki notanda, vinasamlega reynið aftur síðar',
+        )
+      })
+  }
+
+  const refreshList = () => {
+    setIsModalVisible(false)
+    getSupervisors()
+  }
+
+  return (
+    <LoadingContainer
+      isLoading={loading}
+      loader={<ApplicationOverviewSkeleton />}
+    >
+      <Box
+        className={`${headerStyles.header} contentUp delay-25`}
+        marginTop={15}
+        marginBottom={[2, 2, 4]}
+      >
+        <Text as="h1" variant="h1">
+          Umsjónaraðilar
+        </Text>
+        <Button
+          size="small"
+          icon="add"
+          variant="ghost"
+          onClick={() => setIsModalVisible(true)}
+        >
+          Nýr umsjónaraðili
+        </Button>
+      </Box>
+      {supervisors && (
+        <div className={`${tableStyles.wrapper} hideScrollBar`}>
+          <div className={tableStyles.smallTableWrapper}>
+            <table
+              className={cn({
+                [`${tableStyles.tableContainer}`]: true,
+              })}
+            >
+              <thead className={`contentUp delay-50`}>
+                <tr>
+                  {['Nafn', 'Kennitala', 'Hlutverk', 'Aðgerð'].map(
+                    (item, index) => (
+                      <TableHeaders
+                        header={{ title: item }}
+                        index={index}
+                        key={`tableHeaders-${index}`}
+                      />
+                    ),
+                  )}
+                </tr>
+              </thead>
+
+              <tbody className={tableStyles.tableBody}>
```

Comment:
This comes pretty weird when header transition up however the body doesn't, so either place contentUp on the table or place contentUp on the tbody also 🎨